sdk/scheduler/src/test/resources/test-scale-pod-a.yml

```diff
@@ -0,0 +1,36 @@
+name: "test"
+scheduler:
+  api-port: 8080
+pods:
+  pod-a:
+    count: 2
+    resource-sets:
+      resource-set-a:
+        cpus: 1
+        memory: 1000
+        volumes:
```

Comment:
If you want you could also just use `volume:` in single-blind cases like these, see e.g. https://github.com/mesosphere/dcos-commons/blob/master/frameworks/hdfs/src/main/dist/svc.yml#L18